airgun/views/repository.py

```diff
@@ -242,6 +269,15 @@ class OstreeRepository(View):
         class NoSpecificHttpProxy(View):
             pass
 
+    @repo_content.register('file')
+    class FileRepository(View):
+        upstream_url = EditableEntry(name='Upstream URL')
+        verify_ssl = EditableEntryCheckbox(name='Verify SSL')
+        upstream_authorization = AuthorizationEntry(name='Upstream Authorization')
+        publish_via_http = EditableEntryCheckbox(name='Publish via HTTP')
+        unprotected = EditableEntryCheckbox(name='Unprotected')
+        mirroring_policy = EditableEntrySelect(name='Mirroring Policy')
```

Comment:
I see good duplication of the same properties in all types of Repository classes! I believe we should move these all to the common parent/higher class!

The comment is non-blocking and can be a future improvement comment!